sql/rpl_rli.cc

```diff
@@ -2161,6 +2169,7 @@ rpl_group_info::reinit(Relay_log_info *rli)
   start_alter_ev= NULL;
   direct_commit_alter= false;
   commit_orderer.reinit();
+  thr_timer_init(&slave_query_timer, (void (*)(void*)) rgi_query_timeout, this);
```

Comment:
We seem can do without this new initialization as well, please read on about reusing THD's method.